### PR TITLE
Feat/event categories

### PR DESCRIPTION
**Pull request by tuev**

### CODE REVIEW

File: src/Components/Categories/CategoriesArrows.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import classNames from 'classnames'
+import {
+  faChevronRight,
+  faChevronLeft
+} from '@fortawesome/free-solid-svg-icons'
+import './categories.scss'
+
+const Arrows = ({ className, style, onClick, icon, arrow }) => (
+  <div
+    className={classNames(
```
Comment (tuev) [inline src/Components/Categories/CategoriesArrows.js]: classNames method should be moved outside the return  method

File: src/Components/Categories/CategoriesArrows.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      className,
+      ' categories-event__arrow',
+      {
+        'categories-event__arrow--prev': arrow === 'left',
+        'categories-event__arrow--next': arrow === 'right'
+      }
+    )}
+    style={{ ...style }}
```
Comment (tuev) [inline src/Components/Categories/CategoriesArrows.js]: shouldn't pass style as a prop into child component

File: src/Components/Categories/index.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+import 'slick-carousel/slick/slick.css'
+import 'slick-carousel/slick/slick-theme.css'
+import { Container } from 'reactstrap'
+import items from './categories.json'
+import CategoriesItem from './CategoriesItem'
+import { NextArrow, PrevArrow } from './CategoriesArrows'
+import './categories.scss'
+
+export default class Responsive extends Component {
+  render () {
```
Comment (tuev) [inline src/Components/Categories/index.js]: carousel config should be moved to the other fie for readability

File: src/Components/Categories/CategoriesArrows.js (hunk 1 above)
Comment (vuquangit) [reply src/Components/Categories/CategoriesArrows.js]: ok i will fix

File: src/Components/Categories/CategoriesArrows.js (hunk 2 above)
Comment (vuquangit) [reply src/Components/Categories/CategoriesArrows.js]: please help me modify it

File: src/Components/Categories/index.js (hunk 3 above)
Comment (vuquangit) [reply src/Components/Categories/index.js]: ok i will modify it

File: src/Components/Categories/CategoriesArrows.js (hunk 2 above)
Comment (vuquangit) [reply src/Components/Categories/CategoriesArrows.js]: i have fixed it. tks

**Pull request merged**